docs/cel2/l2-operator-guide.md

```diff
@@ -24,61 +24,30 @@ Therefore, operators looking to run full archive nodes or serve requests for his
 Since the Celo L2 node does still require the full pre-migration chain data, these operators will require approximately double the storage space as is currently needed.
 
 :::warning
-The instructions below are not yet fully usable, as some values need to be set. They are marked as TODO.
+The instructions below are not yet fully useable, as some values need to be set.
 :::
 
-## Stopping an L1 Node
+### Starting an L2 node
 
-If you are currently running an L1 Alfajores node, you should upgrade to the below version before the migration and add the `--l2migrationblock` flag when restarting.
-
-:::warning
-TODO: Stay tuned for specific docker image and migration block number
-:::
-
-This will automatically prevent your node from processing blocks higher than `l2migrationblock - 1`.
-
-## Starting an L2 node
-
-### Overview
-
-Three components are required to run an L2 node:
-
-- [eigenda-proxy](https://github.com/Layr-Labs/eigenda-proxy/tree/main): proxy interface between EigenDA and the OP stack
-- op-node: the consensus layer
-- op-geth: the execution layer
-
-For op-geth, there are three options to get started, ranked by ease and the level of trust required:
+Node operators who wish to run an L2 node have three options, ranked by ease and the level of trust required:
 
 1. Start an L2 node with snap sync. This option does not require running the migration script.
 2. Start an L2 node with the provided L1 chaindata.
 3. Migrate the L1 chaindata manually.
 
-Using snap sync offers a simpler and faster experience, but it cannot be used if you want to run an archive node. In that case, you will need to use an archive node snapshot or migrate your own archive data from an L1 node.
+Snap sync provides a simpler and faster setup experience. However, it is not suitable if you plan to run an archive node. In that case, you'll need to either use an archive node snapshot or migrate your own archive data from an L1 node. While both options follow a similar process, there are some differences, particularly in how you prepare the chaindata from the L1. Nonetheless, the majority of the service configuration remains the same across all options.
 
-:::info
```

Comment:
I would keep the extra indication of time for running the migration in the overview and close to the top to avoid surprises.

Reply:
👍